package-lock.json

```diff
@@ -1,9190 +1,16 @@
 {
   "name": "findologic",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
This looks more like a downgrade than an upgrade. Here are my local versions:

* NPM: `7.18.1`
* Node: `v14.15.4`

Try to reset these changes and try again installing those.

Reply:
My node got updated apparently. After downgrading Node to `v14.15.4` stuff works again. Reverting the commit